kafka/getmetrics_kafka.py

```diff
@@ -361,7 +384,12 @@ def filter(self, record):
         # non-zero return means we log this message
         return 1 if record.levelno < self.max_level else 0
 
-
+def add_valuemap(header_field, metric_value, valueMap):
```

Comment:
can you add comments on the purpose of this function?

Reply:
added